middleware/node.go

```diff
@@ -100,14 +121,71 @@ func (n *Node[T]) processDelivery(d Delivery) error {
 }
 
 func (n *Node[T]) freeResources(clientID int, h T) {
-	log.Infof("Cleaning resources for client %v", clientID)
+	log.Infof("Freeing resources for client %v", clientID)
 	err := h.Free()
 	if err != nil {
 		log.Errorf("Error freeing handler files: %v", err)
 	}
 	delete(n.clients, clientID)
 }
 
+func (n *Node[T]) sendAlive(ctx context.Context) error {
+	udpAddr, err := net.ResolveUDPAddr("udp", n.config.Address)
+	if err != nil {
+		return fmt.Errorf("Failed to resolve address: %v", err)
+	}
+
+	conn, err := net.ListenUDP("udp", udpAddr)
```

Comment:
Por ejemplo algo asi (pero con una constante y no un literal):
```suggestion
	conn, err := net.ListenUDP("udp", ":7000")
```